api/queries_pr.go

```diff
@@ -1149,6 +1152,20 @@ func BranchDeleteRemote(client *Client, repo ghrepo.Interface, branch string) er
 	return client.REST(repo.RepoHost(), "DELETE", path, nil, nil)
 }
 
+// sortPRsByPrecedence in the following order: OPEN, MERGED, CLOSED
```

Comment:
If there is a neater way of doing this (for example in the GraphQL query) then I am all ears! 👂

Reply:
Our GraphQL API doesn't allow sorting by state, so doing it in memory like this is fine!